web/source/osk/banner.ts

```diff
@@ -320,7 +320,10 @@ namespace com.keyman.osk {
       if(suggestion.displayAs == null || suggestion.displayAs == '') {
         suggestionText = '\xa0';  // default:  nbsp.
       } else {
-        suggestionText = suggestion.displayAs;
+        // Default the LTR ordering to match that of the active keyboard.
+        let rtl = keyman.keyboardManager.isRTL();
+        let orderCode = rtl ? 0x202e /* RTL */ : 0x202d /* LTR */;
```

Comment:
Why does l. 332 use `\u200f` for rtl marker?

Reply:
Huh.  It was supposed to accomplish a similar goal, but obviously it didn't quite get the RTL job done.  I should probably remove that section in favor of the newer one.